ccd-definition/CaseField.json

```diff
@@ -426,6 +426,55 @@
     "ID": "hearingDate",
     "Label": "The next hearing is on ${hearingDetails.startDate}.",
     "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Comment": "safe to be removed?"
```

Comment:
from what I understand this isn't shown anywhere at the moment? but would old cases with this case field still break?

Reply:
Yeah this is my question. If this gets populated in anyway, then when I remove from the definition, it COULD error. I will ask CCD